presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -1600,4 +1602,34 @@ protocol::NodeStatus PrestoServer::fetchNodeStatus() {
   return nodeStatus;
 }
 
+void PrestoServer::registerDynamicFunctions() {
+  auto systemConfig = SystemConfig::instance();
+  std::error_code ec;
+  const fs::path configPath(systemConfig->dynamicLibraryValidatorConfig());
+  if (!fs::exists(configPath, ec)) {
+    LOG(ERROR) << "Config file not found in path: " << configPath
+               << ". Dynamic libraries will not be loaded on this worker run.";
+    return;
+  }
+  if (fs::is_empty(configPath)) {
```

Comment:
The check for empty config should be before the exists check. We should not call fs::exists on an empty path.